### ISSUE TITLE
f21 server: generated kickstart invalid raid level

### ISSUE THREAD

**Issue by FlorianLudwig** [rhinstaller/anaconda#71]
anaconda generates kickstart files like this:

```
   raid pv.4059 --device=pv00 --fstype="lvmpv" --level=raid1 raid.4047 raid.4053
```

Where `--level=raid1` is not valid but `--level=1`.

**Comment by vpodzime** [comment]
@mulkieran, can you please have a look at this?

**Comment by mulkieran** [comment]
@FlorianLudwig what is the precise problem you experience with the kickstart file? Can you please confirm that ksvalidator rejects the kickstart file?

**Comment by FlorianLudwig** [comment]
@mulkieran yes ksvalidator rejects the file with "RAID Partition defined without RAID level"

The exact problem is `--level=raid1` should be `--level=1` as noted above.

**Comment by mulkieran** [comment]
@FlorianLudwig  Please try "RAID1" instead of "raid1" and let me know what ksvalidator thinks of that.
Thanks!

**Comment by FlorianLudwig** [comment]
@mulkieran yeah, that works

**Comment by mulkieran** [comment]
@FlorianLudwig  Could you confirm that ksvalidator also reject kickstart with "--level=1"? If not, something surprising is happening. Thanks.

**Comment by FlorianLudwig** [comment]
@mulkieran nope, I cannot confirm that as `--level=1` works.

**Comment by FlorianLudwig** [comment]
And i would interpret the docs at

https://github.com/rhinstaller/pykickstart/blob/master/docs/kickstart-docs.rst#raid

`--level=1` being correct.

**Comment by mulkieran** [comment]
Phew, that comma was a colon. So, it makes sense that "1" is a valid specifier for a raid level.

Unfortunately, it's not clear whether this issue is a pykickstart issue or a blivet one yet.

**Comment by mulkieran** [comment]
This is fixed in pykickstart-2.3-1.
